docassemble/EFSPIntegration/data/questions/any_filing_interview.yml

```diff
@@ -474,31 +493,31 @@ fields:
     datatype: dropdown
     code: |
       tyler_payment_account_options
----
-generic object: ALDocument
-id: filing type
-question: |
-  Filing Type
-fields:
-  - Filing Type: x.filing_type
-    datatype: dropdown
-    code: |
-      sorted(filing_type_options, key=lambda option: option[1])
----
-generic object: ALDocument
-id: filing description
-question: |
-  Filing Description
-fields: 
-  - label: |
-      Filing Description
-      % if not filing_description_datafield.get('isrequired'):
-      (optional)
-      % endif      
-    field: x.filing_description
-    datatype: text
-    required: |
-      filing_description_datafield.get('isrequired', False)
+# ---
+# generic object: ALDocument
+# id: filing type
+# question: |
+#   Filing Type
+# fields:
+#   - Filing Type: x.filing_type
+#     datatype: dropdown
+#     code: |
+#       sorted(filing_type_options, key=lambda option: option[1])
+# ---
+# generic object: ALDocument
+# id: filing description
+# question: |
+#   Filing Description
+# fields: 
+#   - label: |
+#       Filing Description
```

Comment:
Same with the description, it should be optional, but there does need to be the change for people to fill it out?